auctions/models.py

```diff
@@ -28,6 +28,8 @@
                          "client_secret": settings.PAYPAL_CLIENT_SECRET,
                          })
 
+PAYPAL_PAYOUT_RECIPIENT = settings.PAYPAL_PAYOUT_RECIPIENT
```

Comment:
Can we skip this extra assignment and just use `settings.PAYPAL_PAYOUT_RECIPIENT` below?